web/beacon-app/src/components/auth/LandingFooter/LandingFooter.tsx

```diff
@@ -9,10 +9,10 @@ import TwitterIcon from '@/assets/icons/twitterIcon';
 function LandingFooter() {
   const { version: appVersion, revision: gitRevision } = appConfig;
   return (
-    <footer className="bg-footer bg-cover bg-no-repeat text-white ">
-      <div className="pt-72 2xl:pt-80">
+    <footer className="mt-20 bg-footer bg-cover bg-no-repeat text-white sm:mt-0 ">
```

Comment:
I noticed the footer was a little too close to the end of the content on smaller screens, so `mt-20` was added for a little breathing room. The style is removed once the `min-width` is 640px.